docs/coverage-reporter/index.md

```diff
@@ -443,3 +447,15 @@ Follow these instructions to validate that your coverage setup is working correc
     -   Branch name and commit SHA-1 hash corresponding to the CI/CD output
     -   Test coverage report that you're uploading to Codacy
     -   Any other relevant information or screenshots of your setup
+
+## 4. Enabling the Git provider integration {: id="enabling-git-provider-integration"}
+
+After validating that the coverage setup is complete, Codacy recommends that you enable the integration with your Git provider to obtain the coverage status and summaries directly on pull requests.
```

Comment:
```suggestion
After validating that the coverage setup is complete, enable the integration with your Git provider to obtain the coverage status and summaries directly on pull requests.
```
I'm unsure what we mean by "Codacy recommends that" here. Is there a prompt, or is this "our" suggestion? Would it make sense to remove this, so the sentence is more imperative?

Reply:
Here the idea is to encourage the users to activate the Git provider integration features, as they'll obtain more value from Codacy. However, that's not mandatory.

We could write instead something like:

> After validating that the coverage setup is complete, _optionally_ enable the integration [...]

However, this doesn't seem to convey the same urgency or "advantage to the users" as an actual recommendation. :thinking: What are your thoughts?

Reply:
OK, I get it now, and I agree it should be stronger than an `optionally`.

I wonder if we may use `we recommend that`? Or is that too "personal"?

Reply:
The Microsoft Style Guide [recommends against that](https://learn.microsoft.com/en-us/style-guide/grammar/person#avoid-plural-first-person-we-us). But now that I read that section more carefully, I realize that the alternative "Codacy recommends [...]" is also not what the style guide recommends. :sweat_smile: 

Also, we already have lots of existing occurrences of "we recommend" in the docs, and only three "Codacy recommends".

So, for now I'm going to change the sentence to "We recommend" for the sake of consistency, and we'll aim to review this style consistently across the whole docs in a future opportunity. Would you like to create a Jira issue in our backlog for this kind of future review @nicklem?

Reply:
👍 https://codacy.atlassian.net/browse/DOCS-497